workspaces/ballerina/ballerina-extension/src/features/project/cmds/add.ts

```diff
@@ -64,3 +108,28 @@ function activateAddCommand() {
 }
 
 export { activateAddCommand };
+
+// Prompts user to select a package
+async function getPackage(packages: string[]): Promise<string | undefined> {
+    const options: PackageQuickPickItem[] = packages.map((pkg) => {
+        return {
+            label: pkg,
+            path: pkg
+        };
+    });
+    if (options.length === 0) {
+        return undefined;
+    }
+    else if (options.length === 1) {
+        return options[0].path;
+    }
+    let resultItem = await window.showQuickPick<PackageQuickPickItem>(options, {
+                placeHolder: `Select a Package to add the module to`,
+            });
```

Comment:
Instead passing `options` let's directly pass the list of package paths as a string array